<launch>
  <include file="$(find mav_nonlinear_mpc)/launch/mav_nonlinear_mpc.launch">
    <arg name="mav_name" value="vrep_quad" />
    <arg name="uav_num" value="1"/>
    <arg name="use_sim_time" value="true"/>
    <arg name="axis_mapping" value="4 3 1 0 4 5 6 7"/>
  </include>
</launch>